src/main/java/com/bootcamp/demo/FirebaseController.java

```diff
@@ -30,7 +37,11 @@
 @RestController
 @RequestMapping(path = "/firebase", produces = APPLICATION_JSON_VALUE)
 public class FirebaseController {
+    private static final ResponseEntity<Object> SUCCESS_RESPONSE = new ResponseEntity<>(HttpStatus.OK);
+    private static final ResponseEntity<Object> FAILURE_RESPONSE = new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+    private static final Logger LOGGER = Logger.getLogger(FirebaseController.class.getName());
     private Firestore firestoreDB;
+    private final BookingService bookingService = new BookingService();
```

Comment:
Since `BookingService` is a bean, you should not instantiate it directly. Rather, you should inject it. Let me know if more details are needed for this.